### PR TITLE
Feat/um 49 query builder api

### PR DESCRIPTION
**Pull request by briancao**
query event data

### CODE REVIEW

File: components/forms/EventDataForm.js [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    }
+  }, [columns]);
+
+  const handleAddTag = (value, list, setState, resetForm) => {
+    setState({ ...list, [`${value.field}`]: value.value });
+    resetForm();
+  };
+
+  const handleRemoveTag = (value, list, setState) => {
+    const { ...rest } = list;
```
Comment (mikecao) [inline components/forms/EventDataForm.js:102]: Makes no sense to spread the list, just use the list.

File: components/forms/EventDataForm.js [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+    const { ok, data } = await post(`/websites/${websiteId}/eventdata`, params);
+
+    if (!ok) {
+      setMessage(<FormattedMessage id="message.failure" defaultMessage="Something went wrong." />);
+      setData([]);
+    } else {
+      setData(data);
+
+      if (message) {
+        setMessage(null);
```
Comment (mikecao) [inline components/forms/EventDataForm.js:128]: No need to check, just always set it null on success.

**Review by mikecao — APPROVED** [review]
Looks good, just a few minor things.

**Review by essesoul — APPROVED** [review]
wow that's awesome

**Pull request merged**